mage_ai/frontend/interfaces/DataSourceType.ts

```diff
@@ -9,6 +9,7 @@ export enum DataSourceTypeEnum {
   GENERIC = 'generic',
   GOOGLE_CLOUD_STORAGE = 'google_cloud_storage',
   KAFKA = 'kafka',
+  RABBITMQ = 'rabbitmq',
```

Comment:
nit: same. could you sort the names in alphabetical order? same in other places.

Reply:
Noticed DATA_SOURCE_TYPES in same file are not in alpheticall order,  will be sorting them too

Reply:
( respecting generic and source files on top)

Reply:
Fixed on new commit (57b943b2dfdcf4d1ab128bf663facbd64e20beee)